docs/admin/index.md

```diff
@@ -15,4 +15,56 @@ and [API](../reference/api/index.md) docs.
 For any information not strictly contained in these sections, check out our
 [Tutorials](../tutorials/index.md) and [FAQs](../tutorials/faqs.md).
 
+## What is an image, template, devcontainer, or workspace
```

Comment:
This may be best presented as a table as this renders quite awkward.

Also I am not sure if this is best on the frontpage of administration. We have no other content here at all. If we do think this is the most important admin concept, we might want to add a sentence explaining why its so important and what else people can expect on the Administration page

<img width="748" alt="Screenshot 2024-11-24 at 9 10 03 PM" src="https://github.com/user-attachments/assets/df82f803-4167-4a3c-a95e-47e0fa3fecd5">

Reply:
I can try a table on and see how it feels - I'd tried an html definition list and it looked okay, but everything within the html would also have to be html (like links), and that didn't seem worth it. my concern with tables is how long text shows in cells, but it's worth a shot

the hope is to have this be a glossary component at some point so that we can define terms in place and have them combined into reference pages

here's an [example in the MyST docs](https://mystmd.org/guide/glossaries-and-terms)

Reply:
it's also interesting that you're still getting served a build of an older commit :-/ not sure why that is

Reply:
adjusted to:

![2024-11-26_14-45-53](https://github.com/user-attachments/assets/71e978e9-7877-47c6-aba6-180a88676d03)